src/routes/todo.routes/getToDos.ts

```diff
@@ -0,0 +1,13 @@
+import { Router, Request, Response } from "express";
+import getToDos from "../../repository/todo.repository/getToDos";
+
+export default function getToDosItems(): Router {
+  const getToDosRouter = Router();
+
+  getToDosRouter.get("/todos/all", async (req: Request, res: Response) => {
+    const todos: [] = (await getToDos()) as [];
```

Comment:
If you define return type in the function itself, casting won't be necessary.
Also, avoid using repository functions inside routers. It's always router -> service -> repository.